xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl

```diff
@@ -113,7 +113,7 @@ spec:
       nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
       nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
       nginx.ingress.kubernetes.io/rewrite-target: /
-      nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
+      nginx.ingress.kubernetes.io/session-cookie-name: ROUTE
```

Comment:
Please check if that is happening for the haproxy, I think we need there change that config and for external too, check lines 129 and 141. Do we need same change on the openshift too?

Reply:
I tried haproxy and it worked as configured already, without changes. openshift was confirmed to be working when the bug was reported by Inba if I recall correctly, does that cover all the bases?

Reply:
haproxy and openshift work as expected with the current configuration :)